pack/parallel/bad_encounter.json

```diff
@@ -0,0 +1,88 @@
+[
```

Comment:
Please check spacing throughout this .json - some of the indentation of lines is off (I've had the same issue myself).